apps/src/templates/sectionProgressV2/progress-table-legend.module.scss

```diff
@@ -30,6 +31,10 @@
   box-sizing: border-box;
 }
 
+.legend.levelTypes {
```

Comment:
Note that having this margin on the last box in the row pushed it over the edge to the next line.  This was my solution to this.  Initially thought about just adding margin to the "middle box" on both the left and right side.  However, after feedback from Liam at one point, I like the idea of `.legend` as being the core style for most of the boxes and that style including padding.

Reply:
I agree with this approach. Give `.legend` a margin by default, that is overridden to zero in this specific case. Looks great!